### PR TITLE
Find out how updating your search page can lead to significant user traffic!

### PR DESCRIPTION
**Pull request by jonathan-best**
screen shot 2016-03-01 at 13 56 09

### CODE REVIEW

File: src/shared/components/NavBar.js [hunk 1]
```diff
@@ -29,36 +21,19 @@ export default class NavBar extends React.Component {
...
-            <li className="navbar__item" key={i++} >
+            <li className="navbar__item">
               <LogoSVG />
             </li>
-            {rows}
+            <li className='pull-right'>
+              <Link
+                style={styles.alpha}
+                to={'/'}>
+                How to guide
```
Comment (DanForys) [inline src/shared/components/NavBar.js]: I think this should be "How-to guide" </pedant>

**Comment by DanForys** [comment]
:+1: to moving more styles into Sass

File: src/shared/components/NavBar.js (hunk 1 above)
Comment (jonathan-best) [inline src/shared/components/NavBar.js]: I'll stick with this for now as its what i have in the design:
https://trello-attachments.s3.amazonaws.com/562a476391737084d56a1162/56d06e9a1dcd75020ab57b7b/78901b451b16a565c5b2295c0363c2ad/homepage_(3).pdf

Can update later when the link and video are available 😊

**Comment by DanForys** [comment]
:cool:

**Pull request merged**